mandatory/4-tax.js

```diff
@@ -17,7 +23,13 @@ function calculateSalesTax() {}
   Remember that the prices must include the sales tax (hint: you already wrote a function for this!)
 */
 
-function addTaxAndFormatCurrency() {}
+function addTaxAndFormatCurrency(productPrice) {
+  let productTax = productPrice * 0.2;
```

Comment:
you could have used the function above, e.g.
`let productTax = calculateSalesTax(productPrice);`